src/App.js

```diff
@@ -1,25 +1,100 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Slider, Select, MenuItem, AppBar, Tabs, Tab } from '@material-ui/core';
 import './App.css';
 
+//const displayCategoryName = event => alert(event.target.id);
+/*
+const categories = [
+  "Наличные деньги",
+  "Кредитные карты",
+  "Ипотека",
+  "Автокредит",
+  "Бизнес кредит"
+];*/
+
 function App() {
+  //Объявляем новую переменную состояния value
+  const [value, setValue]=React.useState(0);
```

Comment:
`setValue` is too generic. Take more specific name please

Reply:
setValue was replaced with setLoan